aptos-move/framework/aptos-framework/sources/TypeInfo.move

```diff
@@ -1,5 +1,5 @@
 module AptosFramework::TypeInfo {
-    struct TypeInfo has copy, drop {
```

Comment:
do we need 'store'? I think Typeinfo should have store by default if it is top level `struct TypeInfo`

Reply:
It's required. Compiler complains otherwise (RegisterEvents: Invalid field type. The struct was declared with the ability 'store' so all fields require the ability 'store')